researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/ResearchStackDAO.java

```diff
@@ -0,0 +1,28 @@
+package org.sagebionetworks.bridge.researchstack;
+
+import android.content.Context;
+
+import org.researchstack.backbone.model.User;
+import org.sagebionetworks.bridge.android.manager.dao.SharedPreferencesJsonDAO;
+
+/**
+ * Created by jyliu on 2/9/2017.
+ */
+
+public class ResearchStackDAO extends SharedPreferencesJsonDAO {
```

Comment:
More on the User issue. So I want to do what's correct, but we should clarify the meaning and use cases for Session & SignIn & StudyParticipant (Bridge concepts) and User.

My thinking when I was working on this is that User mostly duplicates what's already in AuthManager and StudyParticipant, adding in birth date. And then, it seems the use of birth date is while signing consent, if you've already populated a birth date by signing a consent?

It looks like you most recently touched the User stuff, could you clarify what it's for?